canarytokens/channel_input_wireguard.py

```diff
@@ -142,7 +143,7 @@ def datagramReceived(
         # TODO: If canarydrop no longer exists, delete key -> canarytoken mapping in WireGuard keymap
         canarydrop: Optional[Canarydrop] = queries.get_canarydrop(canarytoken)
         if not canarydrop:
-            raise ValueError("Canarydrop not found")
+            raise NoCanarytokenFound
```

Comment:
Two things here:
1. `NoCanarytokenFound` is for [finding tokens in strings](https://github.com/thinkst/canarytokens/blob/master/canarytokens/tokens.py#L110), rather than loading the drops. This is why I'm in favour of a blanket rename of `NoCanarytokenPresent` (which is for [loading drops](https://github.com/thinkst/canarytokens/blob/master/canarytokens/queries.py#L51)) to `NoCanarydropFound` to avoid this confusion...
2. `queries.get_canarydrop(canarytoken)` in the line above is going to raise the error we want here or raise a ValidationError if it fails to parse the drop, so I'm not sure this `if` is needed

Reply:
Thanks @wleightond. That makes sense. Perhaps instead of a blanket change, we make the exception name more understandable?

Reply:
@jayjb what would you suggest?